source/common/json/json_internal.h

```diff
@@ -24,6 +26,11 @@ class Factory {
    */
   static absl::StatusOr<ObjectSharedPtr> loadFromStringNoThrow(const std::string& json);
 
+  /**
+   * Constructs a Json Object from a string.
+   */
```

Comment:
from a protobuf struct.

Reply:
done

Reply:
did you forget to push your latest commit? I ddn't see the change?

Reply:
No just a different file, fixed this one too, thanks